src/main/java/Duke.java

```diff
@@ -1,10 +1,39 @@
+import java.util.Scanner;  // Import the Scanner class
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+
+    public static void intro()
+    {
+        // Logo generated using http://patorjk.com/software/taag/#p=display&f=Fire%20Font-s&t=NUSBOT
+        String logo = "    )       (           )          \n"
+                + " ( /(       )\\ )  (  ( /(   *   )  \n"
+                + " )\\())   ( (()/(( )\\ )\\())` )  /(  \n"
+                + "((_)\\    )\\ /(_))((_|(_)\\  ( )(_)) \n"
+                + " _((_)_ ((_|_))((_)_  ((_)(_(_())  \n"
+                + "| \\| | | | / __|| _ )/ _ \\|_   _|  \n"
+                + "| .` | |_| \\__ \\| _ \\ (_) | | |    \n"
+                + "|_|\\_|\\___/|___/|___/\\___/  |_|    \n"
+                + "                                   \n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Type 'bye' to leave at any time.");
+    }
+
+    public static void echoInput(String input) {
+        System.out.println(input);
+    }
+
+
+    public static void main(String[] args) {
+        intro();
+
+        // Prepre for first input
+        Scanner inputScanner = new Scanner(System.in);
+        System.out.println("What can I do for you?");
+        String userInput = inputScanner.nextLine();
+
+        while(args[0] != "bye" || args[0] != "Bye") {
```

Comment:
I feel that it is a good effort to add comments to help the readers follow your code. However, I feel that it may not be necessary to add the comment in line 1. This is because it does not really help the reader to understand the code better as there is no logic that is difficult to understand.

I liked how you put large chunks of code doing the same thing in a method. It made reading your code a lot easier and cleaner.

At line 34, you could consider using ".equals" instead of "=" as we are dealing with strings.

Reply:
Hi, thanks for the review. I think this is an outdated version of the file, which doesn't include much code. You can find the most recent versions of the files in the pull request here: https://github.com/nus-cs2113-AY1920S2/duke/pull/13/files